README.md

```diff
@@ -0,0 +1,175 @@
+# GitOps Secrets for Node.js
+
+Hello, GitOps Secrets! Goodbye environment variable storage limits.
+
+![GitOps SecretsDiagram](https://user-images.githubusercontent.com/133014/158977309-ce9efc17-ba94-4cb7-a7a4-bdb101a67e6d.jpg)
+
+It's been a long-standing frustration that AWS Lambda deployments have a 4KB environment variable limit. This limit also impacts other environments such as [Vercel](https://vercel.com/support/articles/how-do-i-workaround-vercel-s-4-kb-environment-variables-limit) and the [Serverless framework](https://www.serverless.com/framework/docs/providers/aws/guide/variables) who use AWS Lambda as their infrastructure provider.
+
+A GitOps Secrets workflow eliminates environment variable limits without insecure hacks such as storing unencrypted .env files in your builds and only takes three steps:
+
+1. Install the GitOps Secrets package (currently in developer preview):
+
+```sh
+npm install git+https://github.com/DopplerUniversity/gitops-secrets-nodejs.git
+```
+
+2. Bundle encrypted secrets into your build
+
+```js
+// ./bin/encrypt-secrets.js
+const secrets = require("gitops-secrets");
+
+async function main() {
+  const payload = await secrets.providers.doppler.fetch();
+  secrets.build(payload);
+}
+
+main();
+```
+
+```js
+{
```

Comment:
This is probably fairly clear already but a `// package.json` comment might be nice here